sys/net/gnrc/Kconfig

```diff
@@ -10,5 +10,10 @@ menu "GNRC Network stack"
 rsource "link_layer/lorawan/Kconfig"
 rsource "network_layer/ipv6/blacklist/Kconfig"
 rsource "network_layer/ipv6/whitelist/Kconfig"
+rsource "network_layer/sixlowpan/Kconfig"
+rsource "network_layer/sixlowpan/frag/fb/Kconfig"
+rsource "network_layer/sixlowpan/frag/rb/Kconfig"
+rsource "network_layer/sixlowpan/frag/vrb/Kconfig"
+rsource "network_layer/sixlowpan/nd/Kconfig"
```

Comment:
What about including all these from `sixlowpan/Kconfig`? Also `sixlowpan/frag/*/Kconfig` can be included from a `sixlowpan/frag/Kconfig`.